### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by somokol** [Citrinate/gleamSolver#7]
Is there a method you can add that auto-fills the date of birth fields that some contests require?

Here's a link to a contest that has a date of birth field requirement, for example.

https://gleam.io/MorB4/mobile-strike-4k-hdtv-contest

Thanks again for all your excellent work creating this script!

**Comment by Citrinate** [comment]
I don't believe I can interact with Firefox or Chrome's auto-complete features.  It's possible though to implement my own simple auto-complete feature just for the birthday input.

I like the idea, but I'm not sure when I'll get around to doing it.
